extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/spi/QuarkusClassloadingService.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.smallrye.graphql.runtime.spi;
+
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+
+import graphql.schema.PropertyDataFetcherHelper;
+import io.smallrye.graphql.execution.Classes;
+import io.smallrye.graphql.spi.ClassloadingService;
+
+/**
+ * Quarkus specific classloading service, that allows
+ * hot reloading to work in dev mode.
+ * 
+ * @author Phillip Kruger (phillip.kruger@redhat.com)
+ */
+public class QuarkusClassloadingService implements ClassloadingService {
```

Comment:
This needs @stuartwdouglas 's attention.

Reply:
Yes, this is the hack until @stuartwdouglas 's fix is in. Same as open-api extension. I'll update both once the fix is in.

Reply:
Can I resolve this ? As I said as soon as there is a fix I will remove the hack here and in open-api ? @gsmet  ?